src/ApiPayloadConverter.js

```diff
@@ -22,6 +22,83 @@ class ApiPayloadConverter {
 
     return payload;
   }
+
+  /**
+   * @description Converts the mail payload for the AWS Simple Mail Service (AWS JavaScript SDK v3).
+   * @param {Object} originalPayload The original payload (provider agnostic).
+   * @returns {Object} The payload according to AWS SDK specification.
+   */
+  static awsSES(originalPayload) {
```

Comment:
```suggestion
  static awsSes(originalPayload) {
```

We use strict camel case, so even abbreviations are only capitalized with first letter.

Reply:
Noted and fixed!